webrtc.html

```diff
@@ -11209,6 +11205,36 @@ <h2>
                     </p>
                   </li>
                 </ol>
+                <p>
+                  The <dfn class="export">codec match</dfn> algorithm given two {{RTCRtpCodec}} or compatible dictionaries <var>first</var> and <var>second</var> is as follows:
+                </p>
+                <ol class=algorithm>
+                  <li>
+                    <p>
+                      If <var>first</var>.{{RTCRtpCodec/mimeType}} is different from <var>second</var>.{{RTCRtpCodec/mimeType}}, return <code>false</code>.
```

Comment:
Sorry this comment should go on `channels` and `sdpFmtpLine` which are not required like mimeType is.

Reply:
But we still need to test for existence of both unless we break this out into specific video and audio comparisons

Reply:
Obviously, we want to make sure the dictionaries are similar, and if someone added a `channels` value on a video codec, it should be rejected since no codec returned by `getCapabilities('video')` would have it.
Done!